### PR TITLE
Resolve alignment on the "people" cards

### PR DESCRIPTION
**Pull request by NPDebs**
This is related to PR #499.
It fixes issue #432.

This was resolved by adding a fixed height (230px) to the `media` class, and 150px to the `people-description` class. 

Screenshot (420)

**Note for reviewers:** This does leave some whitespace, but that might be a sacrifice worth paying for proper alignment. :)

### CODE REVIEW

File: _layouts/people.html [hunk 1]
```diff
@@ -370,4 +370,143 @@ <h2>Details</h2>
         {% assign ols-roles = ols-roles | remove_first: ' / ' %}
         {% include _includes/people.html username=username user=user ols-roles=ols-roles %}
     {% endfor %}
+  {% for entry in site.data.people %}
```
Comment (bebatut) [inline _layouts/people.html:373]: All changes below are not needed

File: _includes/people.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="card people-card" id="{{ username }}">
   <div class="card-content">
-    <div class="media">
+    <div class="media" style="height: 230px;">
```
Comment (bebatut) [inline _includes/people.html:3]: We would put the height in the CSS maybe with a dedicated class

File: _includes/people.html [hunk 3]
```diff
@@ -51,7 +51,7 @@
     </div>
 
     {% if ols-roles and ols-roles != '' or user.title %}
-    <p class="people-description">
+    <p class="people-description" style="height: 150px;">
```
Comment (bebatut) [inline _includes/people.html:54]: Height in the CSS

File: css/custom.scss [hunk 4]
```diff
@@ -164,6 +164,16 @@ $link-active: $link-hover;
     font-weight: 600;
   }
 
+  .people-details {
```
Comment (bebatut) [inline css/custom.scss:167]: This class is not used. So the changes are not needed

File: _includes/people.html (hunk 2 above)
Comment (NPDebs) [reply _includes/people.html:3]: Done. 👍

File: _includes/people.html (hunk 3 above)
Comment (NPDebs) [reply _includes/people.html:54]: 💯

File: _layouts/people.html (hunk 1 above)
Comment (NPDebs) [reply _layouts/people.html:373]: I think the text editor did that. I'll change that...

Comment (NPDebs) [reply _layouts/people.html:373]: Woah. 😮 This looks like a badly-resolved merge conflict. I'll delete right away!

File: _includes/people.html (hunk 2 above)
Comment (bebatut) [inline _includes/people.html:3]: Can you remove the height here as it is in the CSS? And check everything if fine :smile:

File: _includes/people.html (hunk 3 above)
Comment (bebatut) [inline _includes/people.html:54]: Can you remove the height here as it is in the CSS? And check everything if fine :smile:

File: _includes/people.html (hunk 2 above)
Comment (NPDebs) [reply _includes/people.html:3]: I could have sworn I removed it. How did I miss it? 😭

**Comment by bebatut** [comment]
It looks a bit strange here: 

image

**Comment by NPDebs** [comment]
Yeah. It looks a bit weird on every card with long description.

The `.people .people-description` class needs a height of `auto`.

**Comment by bebatut** [comment]
Looks better. Thanks

**Pull request merged**